README.md

```diff
@@ -188,19 +199,19 @@ func tr(key: L10n) -> String {
 Once the code has been generated by the script, you can use it this way in your Swift code:
 
 ```swift
-let title = L10n.AlertTitle.string
+let title = L10n.LocalizableTable.AlertTitle.localized
 // -> "Title of the Alert"
 
 // Alternative syntax, shorter
-let msg = tr(.AlertMessage)
+let msg = tr(L10n.LocalizableTable.AlertMessage)
```

Comment:
Yes! Good spot! After adding additional overloads, the compiler (and autocomplete) identified the nested type's shorthand case notation.